driver_test.go

```diff
@@ -61,13 +72,22 @@ func (c *Conn) Prepare(query string) (driver.Stmt, error) {
 	return c.PrepareContext(context.Background(), query)
 }
 
+var testNameTag = "testName"
+
 func (d *Driver) Open(name string) (driver.Conn, error) {
+	var testName string
+	for _, p := range strings.Split(name, "&") {
+		if strings.HasPrefix(p, testNameTag) {
+			testName = strings.Split(p, "=")[1]
+		}
+	}
+
```

Comment:
It would be good to panic here if the testName is not found, or we will start getting weird problems where the empty test name is used by multiple tests if one does not pass the test name in the argument.

Reply:
Agree, it is better to explicitly fail and if we don't want to use this for a test then add a flag or something. This way all tests will be using it by default.